src/app/course/course-routing.module.ts

```diff
@@ -54,6 +55,11 @@ const routes = [
         path: ':courseId/practice/category/:categoryId',
         component: PracticeProblemComponent,
         canActivate: [AuthGuard]
+    },
```

Comment:
We decided that this page should not be in the course. It's a general page showing the practice stats. Then we can add course-specific stats to it as well.